packages/ui/src/theme/css/component/inAppMessaging/banner.scss

```diff
@@ -0,0 +1,49 @@
+.amplify-in-app-messaging-banner__button {
+  background-color: var(--amplify-components-bannermessage-button-background-color);
+  border-style: none;
+  width: var(--amplify-components-bannermessage-button-width);
+}
+.amplify-in-app-messaging-banner__buttons-container {
+  margin-top: var(--amplify-components-bannermessage-buttons-container-margin-top);
+}
+.amplify-in-app-messaging-banner__container {
+  background-color: var(--amplify-components-bannermessage-banner-container-background-color);
+  box-shadow: var(--amplify-components-bannermessage-banner-container-box-shadow);
+  flex-direction: column;
+  margin: var(--amplify-components-bannermessage-banner-container-margin);
+  min-width: 400px;
+  padding: var(--amplify-components-bannermessage-banner-container-padding);
+  position: absolute;
+  right: 0;
+  &.xl, &.large, &.medium {
+      max-width: 50%;
+  }
+  &.base, &.small {
+      width: calc(100% - 32px);
+  }
+}
+.amplify-in-app-messaging-banner__content-container {
+  flex-direction: row;
+}
+.amplify-in-app-messaging-banner__image-container {
+  justify-content: 'center';
+  padding: var(--amplify-components-bannermessage-image-container-padding);
+  width: 15%;
+}
+.amplify-in-app-messaging-banner__text-container {
+  flex: 1;
+  flex-direction: column;
+  justify-content: var(--amplify-components-bannermessage-text-container-justify-content);
+}
+.amplify-in-app-messaging-banner-top {
+  top: 0;
+}
+.amplify-in-app-messaging-banner-middle {
+  bottom: 0;
+  height: fit-content;
+  margin: auto;
+  top: 0;
```

Comment:
Couple things I'm just curious about regarding this class:
- do we need the `bottom` and `top` CSS properties? Just thinking maybe the `margin: auto` would suffice
- what is different about this class that requires a `height` value? Noticed it is not applied kn the `bottom` and `top` variants

Reply:
Because the banner is absolutely positioned, an auto margin isn't having an effect without the `top` and `bottom` values. Using the `top` and `bottom` however stretches the element to full page, which is the reason for the `height` value. 

Happy to streamline this positioning if you've got a preferred vertical-alignment method!